federation/pkg/dnsprovider/providers/coredns/rrsets.go

```diff
@@ -75,6 +75,8 @@ func (rrsets ResourceRecordSets) Get(name string) (dnsprovider.ResourceRecordSet
 			rrset.rrsType = rrstype.CNAME
 		case ip.To4() != nil:
 			rrset.rrsType = rrstype.A
+		default:
```

Comment:
Will take a guess at option #2, adding a comment, so that this can move forward.